code/game/mecha/mecha_wreckage.dm

```diff
@@ -80,7 +80,7 @@
 			to_chat(user, span_warning("You don't see anything that can be pried with [I]!"))
 
 
-/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, null, obj/item/aicard/card)
+/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, _, obj/item/aicard/card)
```

Comment:
```suggestion
/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, mob/living/silicon/ai/AI, obj/item/aicard/card)
```

even if we're not gonna use it, might as well have it match the actual proc.

Reply:
That will break the proc, as it references the object level AI variable